examples/CMakeLists.txt

```diff
@@ -5,5 +5,6 @@ add_subdirectory(miral-kiosk)
 add_subdirectory(miral-system-compositor)
 add_subdirectory(mir_demo_server)
 add_subdirectory(mir-x11-kiosk)
+add_subdirectory(miral-custom-decorations)
```

Comment:
I don't think custom decorations need their own example. Adding them as a miral-shell option would be fine